includes/admin/class-wc-payments-admin.php

```diff
@@ -444,6 +444,7 @@ public function add_payments_menu() {
 	 * Register the CSS and JS scripts
 	 */
 	public function register_payments_scripts() {
+		WC_Payments::load_script_with_dependencies( 'WCPAY_DASH_APP', 'dist/index' );
```

Comment:
Nice addition here, but the left-overs are still there.

Reply:
Addressed in c56251cc14c44e5ca4dd8b249a562c53c322afe6